### PR TITLE
#160056172 Set up empty react app and configure linting

### PR DESCRIPTION
**Pull request by Veraclins**
#### What does this PR do?
Set up an empty react app using create react app
#### Description of Task to be completed?
- Setup a basic react app with create-react-app
- Configure Eslint for the application
- Add components folder
#### How should this be manually tested?
- Clone the repo: Run `git clone https://github.com/andela/elven-ah-client.git` in the folder where you want the repo to be saved.
- Navigate into the newly created folder and install the dependencies: `cd elven-ah-client && npm install`
- Start the app using `npm start`
- Open http://localhost:3000 in a browser 
#### What are the relevant pivotal tracker stories? 
[#160056172](https://www.pivotaltracker.com/story/show/160056172)

### CODE REVIEW

**Review by chukwuemekachm — COMMENTED** [review]
Weldone @Veraclins, I see a did a good job by using create-react-app to bootstrap the project. I also love that you took the initiative to add linting to the project so we can all adhere to Airbnb-React. However, you have two EOL squigglies on `package.json` and `.eslintrc.json`.

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (Oluwafayokemi) [inline src/index.js:2]: Hi Clinton! Well done on this task, I think we should be using react-router-dom and not react-dom. Also, I think you should be declaring the router in the entry file- index.js and not in the container. overall you did a good job.

**Review by Oluwafayokemi — COMMENTED** [review]
Well done!

File: src/index.js (hunk 1 above)
Comment (Veraclins) [reply src/index.js:2]: Thanks for noticing that. That is not supposed to be part of this PR. Have fixed that. Thanks

**Review by Oluwafayokemi — COMMENTED** [review]
I failed to point out that there are some files we will not be needing, like the logo.svg, you might want to take them off

**Comment by Veraclins** [comment]
@chukwuemekachm Thanks for your review. I have fixed the end of line issues. Thanks once again

**Comment by Veraclins** [comment]
@Oluwafayokemi, there is practically nothing in the default app that we will use as it is, but I fee that since we will be changing them later and because some of us are new to react,  having them there could give us an idea about how to implement some things. Also, the app will take real structure when redux and router is added.

File: src/index.js (hunk 1 above)
Comment (chukwuemekachm) [reply src/index.js:2]: @Oluwafayokemi, please note that `react-dom` and `react-router-dom` are two separate libraries. The later is used for rendering the app on the browser while the former is used for client-side routing.
I believe the `react-dom` @Veraclins used is appropriate.

**Review by tonymontaro — APPROVED** [review]
Looks good and works as expected!

**Pull request merged**